bot/seasons/evergreen/wikipedia.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from typing import Optional
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch={search_term}&format=json"
+WIKIPEDIA_URL = "https://en.wikipedia.org/wiki/{title}"
+
+
+class WikipediaCog(commands.Cog):
+    """Get info from wikipedia."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.http_session = bot.http_session
+
+    async def search_wikipedia(self, search_term: str) -> Optional[str]:
+        """Search wikipedia and return the first page found."""
+        async with self.http_session.get(SEARCH_API.format(search_term=search_term)) as response:
+            data = await response.json()
+
+        search_result = data["query"]["search"]
+        if len(search_result) == 0:
+            return None
```

Comment:
The returned API does contain the amount of hits, and using this is much more accurate, specially when the API can cut off the amount of results it return, like in this case:

![image](https://user-images.githubusercontent.com/19921765/78028480-498e5580-7389-11ea-9c8e-d32a13ca3dfa.png)

I think showing this number is cool as well, as well as the suggestion that wikipedia suggested!